packages/blade/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,564 @@
+ControlledSingleSelection.storyName = 'Controlled Single Selection';
+ControlledSingleSelection.args = {
+  accessibilityLabel: 'Select Business type:',
+};
+
+const ControlledMultiSelectionTemplate: ComponentStory<typeof ChipGroupComponent> = (args) => {
+  const chipValues = ['Refunded', 'Failed', 'Pending', 'In Progress'];
+  const [values, setValues] = React.useState(['In Progress']);
+  return (
+    <Box display="flex">
+      <Dropdown marginRight="spacing.4">
+        <DropdownButton size="small">Quick Filters:</DropdownButton>
+        <DropdownOverlay>
+          <ActionList>
+            {chipValues.map((chipValue: string) => (
+              <ActionListItem
+                key={chipValue}
+                title={chipValue}
+                value={chipValue}
+                onClick={({ name, value }) =>
+                  value
+                    ? setValues(values.filter((v) => v !== name))
+                    : setValues(values.concat([name]))
+                }
+                isSelected={values.includes(chipValue)}
+              />
+            ))}
+          </ActionList>
+        </DropdownOverlay>
+      </Dropdown>
+      <ChipGroupComponent
+        {...args}
+        selectionType="multiple"
+        onChange={({ values }) => setValues(values)}
+        value={values}
+      >
+        {chipValues.map((chipValue: string) => (
+          <ChipComponent key={chipValue} value={chipValue} icon={args.icon}>
+            {chipValue}
+          </ChipComponent>
+        ))}
+      </ChipGroupComponent>
+    </Box>
+  );
+};
+
+export const ControlledMultiSelection = ControlledMultiSelectionTemplate.bind({});
+ControlledMultiSelection.storyName = 'Controlled Multiple Selection';
+ControlledMultiSelection.args = {
+  accessibilityLabel: 'Quick Filters:',
+};
+
+export const Disabled = ChipTemplate.bind({});
+Disabled.storyName = 'Disabled';
+Disabled.args = {
+  isDisabled: true,
+};
+
+const ChipWithIconTemplate: ComponentStory<typeof ChipGroupComponent> = ({ children, ...args }) => {
+  return (
+    <Box>
+      <Text marginBottom="spacing.3" marginTop="spacing.3">
+        What other capabilities are you looking for?
+      </Text>
+
+      <ChipGroupComponent selectionType="multiple" defaultValue="payment-links" {...args}>
+        <ChipComponent value="payment-links" icon={PaymentLinksIcon}>
+          Automated Payment Links
+        </ChipComponent>
+        <ChipComponent value="wallet" icon={SmartphoneIcon}>
+          Wallet on My App
+        </ChipComponent>
+        <ChipComponent value="offers" icon={OffersIcon}>
+          Offer discounts, Pay Later & EMI options
+        </ChipComponent>
+      </ChipGroupComponent>
+    </Box>
+  );
+};
+
+export const ChipWithIcon = ChipWithIconTemplate.bind({});
+ChipWithIcon.storyName = 'With Icon';
+ChipWithIcon.args = {
+  accessibilityLabel: 'Quick Filters:',
+};
+
+const AllChipSizesTemplate: ComponentStory<typeof ChipGroupComponent> = ({ children, ...args }) => {
```

Comment:
why do we need this? this seems kitchen sink in the middle of stories. we anyway have knobs which are working fine. if you feel we shall only have sizes maybe? but sizes and with/without icons is too much in the centre of the doc

Reply:
Updated.